src/Microsoft.Azure.WebJobs.Host/Hosting/LoggableOptions/LoggerFilterOptionsLoggableWrapper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Hosting
+{
+
+    internal class LoggerFilterOptionsLoggableType : ILoggableOptionsWrapper<LoggerFilterOptions>
```

Comment:
You have to wrap like this because we don't own the LoggerFilterOptions type?

Reply:
We don't own LoggerFilterOptions, correct. There are likely other built-in Options that we want to log as well. Not sure I understand your proposal -- isn't this just a different wrapper type? Are you saying that one, single type would log all the options that we don't own (as opposed to one-wrapper-type-per-options)?

Reply:
Yeah, I was thinking about the same approach.

Reply:
Oh! Sounds like we're on the same page -- I commented above that I'm going to remove `Set()` and just have it be `GetLoggableOptions(TOptions options)`. You're right that this wouldn't be a wrapper anymore... it'd just log as it's passed in, so I'll adjust the naming.